samples/linker-instructions-advanced.md

```diff
@@ -0,0 +1,63 @@
+# Using IL Linker Advanced Features
+
+This document described the more advanced features for the IL Linker and provides more insight into how it functions.
```

Comment:
"More" should be dropped, unless we describe some functionality in the basic document.